src/components/AuctionDetails/AuctionDetails.tsx

```diff
@@ -164,6 +165,15 @@ function AuctionDetails(props: Props) {
             return value;
         }
 
+        if (key === "color") {
+            let decSplits = value ? value.split(":") : [];
+            let hexSplits: string[] = [];
+            decSplits.forEach(split => {
+                hexSplits.push(parseInt(split).toString(16));
+            })
+            return <Tooltip type="hover" content={<span>{hexSplits.join(":")}</span>} tooltipContent={value} />
```

Comment:
```suggestion
            return <Tooltip type="hover" content={<span>{hexSplits.join("")}</span>} tooltipContent={value} />
```
The hex format isn't seperated by colons, the parts are always the same length